DataFormats/HcalDetId/interface/HcalZDCDetId.h

```diff
@@ -1,22 +1,27 @@
-#ifndef HcalZDCDetId_h_included
-#define HcalZDCDetId_h_included 1
+#ifndef DataFormats_HcalDetId_HcalZDCDetId_h_included
+#define DataFormats_HcalDetId_HcalZDCDetId_h_included 1
 
 #include <ostream>
 #include "DataFormats/DetId/interface/DetId.h"
 
 /** \class HcalZDCDetId
   *  
   *  Contents of the HcalZDCDetId :
+  *     [7]   Set for RPD
   *     [6]   Z position (true for positive)
   *     [5:4] Section (EM/HAD/Lumi)
   *     [3:0] Channel
   *
-  * \author J. Mans - Minnesota
   */
 class HcalZDCDetId : public DetId {
 public:
-  enum Section { Unknown=0, EM=1, HAD=2, LUM=3 };
-  // 1 => CaloTower, 3 => Castor
+  static const int kZDCChannelMask   = 0xF;
+  static const int kZDCSectionMask   = 0x3;
+  static const int kZDCSectionOffset = 4;
+  static const int kZDCZsideMask     = 0x40;
+  static const int kZDCRPDMask       = 0x80;
```

Comment:
Please note that "this is needed only for constants used outside of this file or HcalZDCDetId.cc"

```
N3690 (should be C++11 standard) and latest draft N4567, 9.4.2/3

...
The member shall still be defined in a namespace scope if it
is odr-used (3.2) in the program and the namespace scope definition
shall not contain an initializer.
9.4.2/4 talks about how const static data members are being handled.
```